src/api/v1/db/oracledb/setupDb.sql

```diff
@@ -0,0 +1,72 @@
+DROP TABLE PIZZA_INGREDIENTS;
+DROP TABLE PIZZAS;
+DROP TABLE INGREDIENTS;
+DROP TABLE DOUGHS;
+
+CREATE TABLE "DOUGHS" 
+   (	"ID" NUMBER GENERATED ALWAYS AS IDENTITY MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER  NOCYCLE  NOT NULL ENABLE, 
+	"NAME" VARCHAR2(256 BYTE) NOT NULL ENABLE, 
+	"GRAMS_FLOUR" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_WATER" NUMBER(*,0) NOT NULL ENABLE, 
+	"FLOUR_TYPE" VARCHAR2(256 BYTE) NOT NULL ENABLE, 
+	"WATER_TEMP" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_YEAST" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_SALT" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_SUGAR" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_OLIVE_OIL" NUMBER(*,0) NOT NULL ENABLE, 
+	"BULK_FERMENT_TIME" NUMBER(*,0) NOT NULL ENABLE, 
+	"PROOF_TIME" NUMBER(*,0) NOT NULL ENABLE, 
+	"SPECIAL_INSTRUCTIONS" VARCHAR2(512 BYTE), 
+	 CONSTRAINT "DOUGHS_PK" PRIMARY KEY ("ID") ENABLE
+   );
+
+CREATE TABLE "INGREDIENTS" 
+   (	"TYPE" VARCHAR2(20 BYTE) NOT NULL ENABLE, 
+	"NAME" VARCHAR2(256 BYTE) NOT NULL ENABLE, 
+	"NOTES" VARCHAR2(256 BYTE), 
+	"ID" NUMBER GENERATED ALWAYS AS IDENTITY MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER  NOCYCLE  NOT NULL ENABLE, 
+	 CONSTRAINT "INGREDIENTS_PK" PRIMARY KEY ("ID") ENABLE
+   );
+
+CREATE TABLE "PIZZAS" 
+   (	"ID" NUMBER GENERATED ALWAYS AS IDENTITY MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER  NOCYCLE  NOT NULL ENABLE, 
+	"DOUGH_ID" NUMBER, 
+	"NAME" VARCHAR2(256 BYTE) NOT NULL ENABLE, 
+	"BAKE_TIME" NUMBER(*,0), 
+	"SPECIAL_INSTRUCTIONS" VARCHAR2(512 BYTE), 
+	"OVEN_TEMP" NUMBER(*,0) NOT NULL ENABLE, 
+	 CONSTRAINT "PIZZAS_PK" PRIMARY KEY ("ID") ENABLE,
+	 CONSTRAINT "PIZZAS_FK1" FOREIGN KEY ("DOUGH_ID")
+	  REFERENCES "DOUGHS" ("ID") ON DELETE SET NULL ENABLE
+   );
+
+CREATE TABLE "PIZZA_INGREDIENTS" 
+   (	"PIZZA_ID" NUMBER NOT NULL ENABLE, 
+	"INGREDIENT_ID" NUMBER NOT NULL ENABLE, 
+	"DESCRIPTION" VARCHAR2(512 BYTE), 
+	 CONSTRAINT "PIZZA_INGREDIENTS_UK1" UNIQUE ("PIZZA_ID", "INGREDIENT_ID") ENABLE,
```

Comment:
Not having a primary key but requiring the combination of pizza id and ingredient id jumped out to me as the most obvious solution for this table. Not sure it's the best.